modules/02b-cdc-connect-debezium.md

```diff
@@ -0,0 +1,121 @@
+# Module 02 - CDC with Apache Kafka Connect and Debezium
```

Comment:
rename this file as the original one, so that it becomes the main one automatically